modules/local/psirc/index/main.nf

```diff
@@ -0,0 +1,24 @@
+process PSIRC_INDEX {
+    tag "${meta.id}"
+    label 'process_high'
+
+    container "registry.hub.docker.com/bigdatainbiomedicine/psirc"
```

Comment:
not available in conda I assume.
Is there any Dockerfile available for this tool?

Reply:
I will add finding a cleaner solution to my todo list

Reply:
Added #115